src/main/java/macrobase/analysis/summary/Summary.java

```diff
@@ -0,0 +1,7 @@
+package macrobase.analysis.summary;
+
+/**
+ * Represents a summarization result, currently an itemset with a score and weight.
+ */
+public abstract class Summary {
```

Comment:
We could, I just wanted new abstract classes, not to carry anything old with it. We can rename things once the API is finalized.